libs/api/database/src/lib/content/schemas/sections.schema.ts

```diff
@@ -1,14 +1,16 @@
 import { Schema, Prop, SchemaFactory, raw } from '@nestjs/mongoose';
 import { Document } from 'mongoose';
 import { nanoid } from 'nanoid';
-
 import { Section, AuthorsNotePos } from '@dragonfish/shared/models/sections';
 
 @Schema({ timestamps: true, autoIndex: true, collection: 'sections' })
 export class SectionsDocument extends Document implements Section {
     @Prop({ default: () => nanoid() })
     readonly _id: string;
 
+    @Prop({ trim: true, required: true })
+    readonly contentId: string;
```

Comment:
What was the purpose of this change?

Reply:
sections currently don't have any association with the content they're part of, which makes it hard to figure out which section belongs to what work when those sections are removed from the work by the author. this change fixes that